boards/arm/mimxrt1010_evk/doc/index.rst

```diff
@@ -191,6 +191,107 @@ see the following message in the terminal:
 
     Hello World! mimxrt1010_evk
 
+MCUXpresso Config Tool
+======================
+
+A ``mimxrt1010_evk.mex`` file is included. This file was used to generate the clock
+initialization code and can be used as a starting point to tweak the clock configuration.
+This could be useful for different boards that are based on i.MX RT1010.
+
+NOTE: The MCUXpresso Config Tool currently generates a ``.c`` file with the clock configuration.
+Considering options on leveraging this tool in the future to generate a devicetree compatible file.
```

Comment:
My personal feeling is that doesn't sound like a firm commitment, but I won't not block if others are fine with it. @mbolivar-nordic, is that wording ok to you ?

Reply:
I am pasting what was mentioned in the DIscord chat. 
"Like on the pinctrl changes, if we want the larger project to completely transition to DTS on the clock control we should put in an RFC that will establish an agreed on timeline to get this done."

Reply:
So, bottom line:
- this statement is purely about tool leveraging. Which, of course, is fine.
- you don't mention any commitment to conversion to DT. You will apply to such conversion when the project as a whole starts a transition activity to DTS for clock control. Which indeed is not worth mentioning.

So there is no mention about plans of any kind for transitioning clocks to DT.
Anyway, I don't think I have the legitimacy to block on such point. I let others comment if needed.
So please consider this remark solved from my perspective.